Bigtable/src/Admin/V2/Gapic/BigtableInstanceAdminGapicClient.php

```diff
@@ -1491,75 +1593,84 @@ public function updateCluster($name, $serveNodes, array $optionalArgs = [])
+     * @param array  $optionalArgs {
+     *     Optional.
      *
+     *     @type string $name
+     *           The unique name of the instance. Values are of the form
+     *           `projects/{project}/instances/[a-z][a-z0-9\\-]+[a-z0-9]`.
      *     @type int $state
-     *          (`OutputOnly`)
-     *          The current state of the instance.
-     *          For allowed values, use constants defined on {@see \Google\Cloud\Bigtable\Admin\V2\Instance\State}
+     *           (`OutputOnly`)
+     *           The current state of the instance.
+     *           For allowed values, use constants defined on {@see \Google\Cloud\Bigtable\Admin\V2\Instance\State}
+     *     @type int $type
+     *           The type of the instance. Defaults to `PRODUCTION`.
+     *           For allowed values, use constants defined on {@see \Google\Cloud\Bigtable\Admin\V2\Instance\Type}
+     *     @type array $labels
+     *           Labels are a flexible and lightweight mechanism for organizing cloud
+     *           resources into groups that reflect a customer's organizational needs and
+     *           deployment strategies. They can be used to filter resources and aggregate
+     *           metrics.
+     *
+     *           * Label keys must be between 1 and 63 characters long and must conform to
+     *           the regular expression: `[\p{Ll}\p{Lo}][\p{Ll}\p{Lo}\p{N}_-]{0,62}`.
+     *           * Label values must be between 0 and 63 characters long and must conform to
+     *           the regular expression: `[\p{Ll}\p{Lo}\p{N}_-]{0,63}`.
+     *           * No more than 64 labels can be associated with a given resource.
+     *           * Keys and values must both be under 128 bytes.
+     *     @type Timestamp $createTime
+     *           Output only. A server-assigned timestamp representing when this Instance was created.
+     *           For instances created before this field was added (August 2021), this value
+     *           is `seconds: 0, nanos: 1`.
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
      * @return \Google\Cloud\Bigtable\Admin\V2\Instance
      *
      * @throws ApiException if the remote call fails
      */
-    public function updateInstance($name, $displayName, $type, $labels, array $optionalArgs = [])
+    public function updateInstance($displayName, array $optionalArgs = [])
```

Comment:
Required-to-optional parameters. This breaks BC and will need to be fixed in the generator